config/crd/bases/workload.nephio.org_smfdeployments.yaml

```diff
@@ -0,0 +1,366 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.2
+  creationTimestamp: null
+  name: smfdeployments.workload.nephio.org
+spec:
+  group: workload.nephio.org
+  names:
+    kind: SMFDeployment
+    listKind: SMFDeploymentList
+    plural: smfdeployments
+    singular: smfdeployment
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              capacity:
+                description: capacity defines the capacity characteristics of the
+                  NF deployment
+                properties:
+                  maxDownlinkThroughput:
+                    anyOf:
+                    - type: integer
+                    - type: string
+                    description: MaxDownlinkThroughput defines the max downlink dataplane
+                      throughput
+                    pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
+                    x-kubernetes-int-or-string: true
+                  maxNFConnections:
+                    description: MaxNFConnections defines the max NF(s) that can be
+                      connected to this NF/device
+                    type: integer
+                  maxSessions:
+                    description: MaxSessions defines the max sessions of the control
+                      plane expressed in unit of 1000s
+                    type: integer
+                  maxSubscribers:
+                    description: MaxSubscribers defines the max subscribers expressed
+                      in unit of 1000s
+                    type: integer
+                  maxUplinkThroughput:
```

Comment:
Same question as AMF, but i guess that should resolve this one too..

Reply:
it is all the same since we inherit a generic type definition. if you don't use a parameter you can omit it